src/components/fundamentals/icons/medusa-icon/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import IconProps from "../types/icon-type"
+
+const MedusaIcon: React.FC<IconProps> = ({ size = "16", ...attributes }) => {
```

Comment:
very nit:

the size prop is not used for anything here. This is properly fine since we aren't using it anywhere else atm. But we could do something like this so we can use it in different size at a later time:

```suggestion
const MedusaIcon: React.FC<IconProps> = ({ size = "48", ...attributes }) => {
  const width = +size * 0.9375 // width relative to height (from size prop)
  return (
    <svg
      width={width}
      height={size}
      viewBox="0 0 45 48"
      fill="none"
      xmlns="http://www.w3.org/2000/svg"
      {...attributes}
    >
```